sdv/tabular/copulas.py

```diff
@@ -289,9 +289,11 @@ def _fit(self, table_data):
                 Data to be fitted.
         """
         for column in table_data.columns:
-            distribution = self._field_distributions.get(column)
-            if not distribution:
-                self._field_distributions[column] = self._default_distribution
+            if column not in self._field_distributions:
+                # Check if the column is a derived column.
+                column_name = column.replace('.value', '')
+                self._field_distributions[column] = self._field_distributions.get(
```

Comment:
One thing to note is that the `_field_distributions` dictionary will have some field names that match the `RDT` output, and some that match the input. For example, if null columns are created and there is a column 'a', then the dictionary will have 
```
{
    'a': dist,
    'a.is_null': default_dist
}
````
Idk if it makes sense to have them all match the `HyperTransformer` output names or not (ie. keep the `.value` extension)

Reply:
I created #744 to track this question, not completely sure what we should do right now.